src/Transformers/StopWordFilter.php

```diff
@@ -33,9 +35,11 @@ public function __construct(array $stopWords = [])
             $word = preg_quote($word, '/');
         }
 
-        $pattern = sprintf('/\b(%s)\b/u', implode('|', $stopWords));
+        if (!empty($stopWords)) {
```

Comment:
Only adds a pattern if we have words to search for, else we needn't do anything at all :)